play/PlayerRules/Basic Rules.txt

```diff
@@ -418,49 +418,94 @@ You can jump vertically Str/3 meters.
-the hit is considered an Extra-Critical hit, and deals an extra 20 damage. 
+To hit a target, Players must roll a d10 that is over their weapon's miss stat.
+When an enemy is in cover, their cover score is added to your weapon's miss chance. 
+Lets say your normal ranged miss chance is 2, and the enemy is in full cover, your
+miss chance is increased by 4, for a final DC of 6. So a roll of 7-10 would be 
+successful.
 
-Your Luck modifier ((Luck-5)*4) is subtracted from the lower bound of your critical
-hit range and half of your luck modifier is subtracted from the lower bound of your
-extra-critical hit range. All weapons can critical hit except for Fully-Automatic
-weapons (weapons that shoot more than 1 bullet per half-action by default).
+Critical Hits:
+Critical hits are accuracy rolls that land inside your critical hit range (aka. crit chance). 
+If you roll a critical hit, your weapon does +20 damage for that attack (and/or other effects,
+depending on your abilities). By default, everyone starts with a critical hit range of "1", meaning, an
+attack roll of a 10 would be considered a critical hit. But your critical range may 
+not be "1" because your Luck modifies your critical hit range: ((Luck-5)*0.4), round up, is
+added to your critical hit range. Which means, if you have a Luck stat of 6 (and thus,
+by the equation, +1 to your critical hit range), an attack roll of 9-10 would be considered
+a critical hit. 
+All weapons and melee can critical hit except for Fully-Automatic weapons (weapons that 
+shoot more than 1 bullet per Action by default).
 
 Gun Jams:
-On a roll of 00-01, the weapon Jams, necessitating a weapon skill check, DC 60.
-When a player is defending against a hit, they roll Percentiles under their 
-armor in order to apply a Damage Reduction, or to apply damage to their 
-Capacitive Sheilds if the player has them.
+If at any time that you are firing a gun and you roll a weapon accuracy of 1, roll a second d10.
+If that second d10 roll is a 1 or 2, the weapon Jams, necessitating a weapon 
+skill check, DC 60. 
 
-Hip Firing doubles a weapon''s miss chance. It is much smarter to takes one 
-action to Aim, and spend the rest of your shots on firing. Its also a great 
-idea to take an action at the end of your turn to "hunker," and gain a 25% 
+Armor and Getting Hit:
+When a player is defending against a hit, they roll Percentiles under their 
+armor in order to see if their armor blocked the bullet, and from there, how much
+damage they receive. If the player has no armor, they roll for a target 50 Percentile. 
+If the shot/attack rolls above 50%, they receive half-damage. Below 50%, full damage.
+
+Armor consists of 3 concepts, Armor Points, Armor Piercing (AP) level, and % Coverage.
+Starting from the end, Coverage is simply the percentile that you have to roll
+under to see if armor was hit. If the attack misses armor, or rolls over 50%, it does
+half damage. If the attack hits armor: AP level comes into play. 

```

Comment:
Please state explicitly in the armor rules that missing armor does half damage because the bullet is hitting a less vital part of the body. Otherwise this effect is counterintuitive.